modules/campaignion_petition/campaignion_petition_templates/campaignion_petition_templates.features.uuid_node.inc

```diff
@@ -54,6 +54,8 @@ function campaignion_petition_templates_uuid_features_default_content() {
       ),
     ),
   ),
+  'layout' => array(),
+  'layout_background_image' => array(),
```

Comment:
The new fields are missing from `campaignion_donation_type.features.uuid_node.inc` and the thank-you-pages included in `campaignion_petition_templates.features.uuid_node.inc`.

Reply:
added